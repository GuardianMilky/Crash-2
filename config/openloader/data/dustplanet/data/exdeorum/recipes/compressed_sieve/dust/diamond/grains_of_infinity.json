{"type":"forge:conditional","recipes":[{"conditions":[{"type":"forge:mod_loaded","modid":"enderio"}],"recipe":{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/dust"},"mesh":"exdeorum:diamond_mesh","result":"enderio:grains_of_infinity","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.12}}}]}