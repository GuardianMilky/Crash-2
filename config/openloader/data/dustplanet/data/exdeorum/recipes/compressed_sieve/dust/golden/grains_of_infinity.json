{"type":"forge:conditional","recipes":[{"conditions":[{"type":"forge:mod_loaded","modid":"enderio"}],"recipe":{"type":"exdeorum:compressed_sieve","ingredient":{"tag":"exdeorum:compressed/dust"},"mesh":"exdeorum:golden_mesh","result":"enderio:grains_of_infinity","result_amount":{"type":"minecraft:binomial","n":7.0,"p":0.11}}}]}